common/src/main/java/com/box/l10n/mojito/json/ObjectMapper.java

```diff
@@ -31,6 +33,18 @@ public ObjectMapper(ObjectMapper objectMapper) {
     registerGuavaModule();
   }
 
+  public ObjectMapper(SmileFactory smileFactory) {
+    super(smileFactory);
+    registerJavaTimeModule();
+    registerGuavaModule();
+    // Blackbird module uses bytecode generation to further speed up serialization/deserialization
+    registerBlackbirdModule();
```

Comment:
is this relevant only for smile or all object mappers?

Reply:
Blackbird could be used with any object mapper, it's not specific to smile. I added it here as this constructor is only relevant to smile as it passes in a SmileFactory,